### ISSUE TITLE
When wifi drops, second costumes show broken image icon in the sprite library

### ISSUE THREAD

**Issue by BryceLTaylor** [LLK/scratch-gui#2129]
### Expected Behavior

If you lose connectivity shortly after loading a new project, libraries should not show broken image thumbnails

### Actual Behavior

Though the first costumes load, many of the second costumes appear as the broken image thumbnail.

### Steps to Reproduce

Launch a new project
Turn off your wifi
Open the sprite library
Hover over sprites with more than one costume
Note: many of the sprites try to switch to their other costumes but instead show a broken image thumbnail

![librarywifidrop](https://user-images.githubusercontent.com/1088291/40429315-6cef0a2c-5e70-11e8-9ea1-386f3f82fff0.gif)

Ideally it would just not attempt to show a costume that isn't there and stay on the one/ones that are there.

### Operating System and Browser

Chrome, Firefox, Safari on mac

**Comment by sjhuang26** [comment]
I don't think it is necessary to fix this. If this is fixed, there will still be many more broken assets in other places when wifi drops, and eventually, the user will not be able to work until wifi resumes; thus, the user experience won't be improved by much.
